client/layout/universal-navbar-footer/index.tsx

```diff
@@ -209,68 +213,64 @@ const UniversalNavbarFooter = () => {
 									className="lp-language-picker__content"
 									title={ translate( 'Change Language' ) }
 									onChange={ ( e ) => navigate( e.target.value ) }
-									defaultValue={
-										translate.localeSlug === 'en'
-											? `/plugins`
-											: `/${ translate.localeSlug }/plugins`
-									}
+									defaultValue={ currentRoute }
 								>
 									<option>{ translate( 'Change Language' ) }</option>
-									<option lang="es" value="/es/plugins">
+									<option lang="es" value={ `/es/${ realPathName }` }>
```

Comment:
I'd suggest using the [localizePath](https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/i18n-utils/localize-path.js#L35) utility from `'calypso/lib/i18n-utils/localize-path'`: it ensure that any changes to the URL structure will be reflected in the language changer
```suggestion
									<option lang="es" value={ localizePath( realPathName, 'es', false ) }>
```

Reply:
@claudiucelfilip  - for some reason I see this error, when I click on the above link:

`The 'Automattic/wp-calypso' repository doesn't contain the 'client/lib/i18n-utils/localize-path.js' path in 'trunk'.  `